docs/ecosystem-roadmap.md

```diff
@@ -10,92 +10,108 @@ ms.prod: console
 
 ### Windows Terminal
 
-This is the new Windows interface for command-line applications, intended as a first-party example of using the [pseudoconsole](pseudoconsoles.md) to separate the concerns between API servicing and text-based application interfacing, much like all non-Windows platforms. It is intended to be the flagship text mode user interface for Windows and demonstrate the capabilities of the ecosystem, drive Windows development toward a unification with other platforms, and be an example of how to build a robust and complex modern application that spans the history and gamut of Windows APIs and frameworks. Per the definitions above, this product operates in a terminal role.
+This is the new Windows interface for command-line applications.  Windows Terminal serves as a first-party example of using the [pseudoconsole](pseudoconsoles.md) to separate the concerns between API servicing and text-based application interfacing, much like all non-Windows platforms.
 
-## Roadmap
+Windows Terminal is the flagship text-mode user interface for Windows. It demonstrates the capabilities of the ecosystem and is driving Windows development toward unifying with other platforms. Windows Terminal is also an example of how to build a robust and complex modern application that spans the history and gamut of Windows APIs and frameworks. Per the definitions above, this product operates in a terminal role.
 
-This roadmap starts with a brief history of major milestones in the console subsystem prior to 2014. It then moves into an overview of work performed since 2014 when the renewed focus on the command-line was formed in the Windows 10 era and finishes with what is intended to come next.
+## Major historical milestones
 
-### Implementation
+The major historical milestones for the console subsystem are broken into implementation prior to 2014 and then moves into an overview of work performed since 2014, when the renewed focus on the command-line was formed in the Windows 10 era.
 
-**\[1989-1990s]** The initial console host system was implemented as an emulation of the DOS environment within the Windows operating system. Its code is entangled and cooperative with the [Command Prompt](https://docs.microsoft.com/windows-server/administration/windows-commands/cmd), `cmd.exe`, that is a representation of that DOS environment and shares responsibilities and privileges with that interpreter/shell. It also provides a base level of services for other command-line utilities to perform services in a CMD-like manner.
+### Implementation prior to 2014

```

Comment:
```suggestion
### Initial Implementation
```

Unless you intended to classify this paragraph, DBCS, and Security under a "before 2014" banner.
Then have a banner for 2014-2019.
Then have a banner for "future" in 3 different parts. 

That isn't what really happened here, though, so I'm not sure what you were thinking and I want us to be on the same page.